src/core/namespace.c

```diff
@@ -83,7 +83,6 @@ static const TargetMount protect_kernel_tunables_table[] = {
         { "/proc/sysrq-trigger",        READONLY,       true  },
         { "/proc/latency_stats",        READONLY,       true  },
         { "/proc/mtrr",                 READONLY,       true  },
-        { "/proc/apm",                  READONLY,       true  },
```

Comment:
I don't think we should drop this. APM is upstream in the kernel (as of kernel 4.9) and it's entirely possible to have a modern kernel with this configured (and people do run modern kernels on old hardware). Since it's old and infrequently tested, guarding this is all the more important.